### PR TITLE
[ElasticSearch] Init bundle

### PR DESCRIPTION
**Pull request by Arminek**

### CODE REVIEW

File: src/LakionSyliusElasticSearchBundle.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+use Sylius\Bundle\ResourceBundle\SyliusResourceBundle;
+
+final class LakionSyliusElasticSearchBundle extends AbstractResourceBundle
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getSupportedDrivers()
+    {
+        return [SyliusResourceBundle::DRIVER_DOCTRINE_ORM];
```
Comment (pjedrzejewski) [inline src/LakionSyliusElasticSearchBundle.php]: Not really! :) We won't support drivers other than ORM + ElasticSearch.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            "email": "arkadiusz.krakowiak@lakion.com",
+            "homepage": "http://lakion.com"
+        }
+    ],
+    "require": {
+        "php": "^5.6|^7.0",
+
+        "sylius/sylius": "dev-master",
+
+        "doctrine/phpcr-odm": "^2.0@dev",
```
Comment (pjedrzejewski) [inline composer.json]: Why?

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "friendsofsymfony/elastica-bundle": "^3.2"
+    },
+    "require-dev": {
+        "jackalope/jackalope-doctrine-dbal": "^1.2",
```
Comment (pjedrzejewski) [inline composer.json]: Don't think so. :)

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    - phpenv config-rm xdebug.ini || true
+    - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+    - composer self-update
+
+install:
+    - composer update --prefer-dist
+
+    - src/Tests/Application/bin/console doctrine:schema:create --env=test -vvv --no-interaction
+    - src/Tests/Application/bin/console doctrine:phpcr:repository:init --env=test -vvv --no-interaction
```
Comment (pjedrzejewski) [inline .travis.yml]: ;)

**Review by pjedrzejewski — CHANGES_REQUESTED** [review]

**Comment by pjedrzejewski** [comment]
Thank you Arek!

File: src/DependencyInjection/LakionSyliusElasticSearchExtension.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Lakion\SyliusElasticSearchBundle\DependencyInjection;
+
+use Sylius\Bundle\ResourceBundle\DependencyInjection\Extension\AbstractResourceExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class LakionSyliusElasticSearchExtension extends AbstractResourceExtension
```
Comment (pamil) [inline src/DependencyInjection/LakionSyliusElasticSearchExtension.php:8]: I guess this extension does not need to extend `AbstractResourceExtension`.

File: src/LakionSyliusElasticSearchBundle.php [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Lakion\SyliusElasticSearchBundle;
+
+use Sylius\Bundle\ResourceBundle\AbstractResourceBundle;
+use Sylius\Bundle\ResourceBundle\SyliusResourceBundle;
+
+final class LakionSyliusElasticSearchBundle extends AbstractResourceBundle
```
Comment (pamil) [inline src/LakionSyliusElasticSearchBundle.php:8]: This bundle does not need to extend `AbstractResourceBundle`.

File: src/Tests/Application/app/AppKernel.php [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+final class AppKernel extends Kernel
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function registerBundles()
+    {
+        return array_merge([
+            new \Lakion\SyliusElasticSearchBundle\LakionSyliusElasticSearchBundle(),
```
Comment (pamil) [inline src/Tests/Application/app/AppKernel.php:14]: It probably can be placed together with other bundles, no need to include it as the first bundle.

File: src/Tests/Application/app/config/config.yml [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+doctrine:
+    dbal:
+        driver: "pdo_sqlite"
+        path: "%kernel.cache_dir%/db.sql"
+        charset: UTF8
+
+sylius_resource:
+    drivers:
+        - doctrine/orm
```
Comment (pamil) [inline src/Tests/Application/app/config/config.yml:38]: Is it really needed?

File: src/Tests/Application/app/config/config.yml [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+sylius_grid:
+    drivers:
+        - doctrine/orm
```
Comment (pamil) [inline src/Tests/Application/app/config/config.yml:42]: Is it really needed?

**Pull request merged**